code/modules/clothing/gloves/color.dm

```diff
@@ -8,6 +8,22 @@
 	item_color="yellow"
 	resistance_flags = NONE
 
+/obj/item/clothing/gloves/color/yellow/equipped(mob/user, slot)
+	. = ..()
+	if(slot == SLOT_GLOVES)
+		if(user.mind && user.mind.assigned_role == "Assistant")
+			SEND_SIGNAL(user, COMSIG_ADD_MOOD_EVENT, "assistant_insulated_gloves", /datum/mood_event/assistant_insulated_gloves)
+		if(user.mind && user.mind.assigned_role == "Security Officer")
```

Comment:
```suggestion
		if(user.mind && user.mind.assigned_role == "Security Officer" && item_color=="yellow")
```

@githubuser343 would this work?

Reply:
After a brief look over washing machine and crayon code, yes that's how it seems to work.

Reply:
I can reason not giving black colored insuls a _buff_ by "These don't feel like my normal gloves".

This PR gives the most minor buff to Security that play by the rules, I don't really want to reward people that cheese it.